indico/modules/events/papers/controllers/management.py

```diff
@@ -219,3 +214,86 @@ def _process(self):
             flash(messages[role], 'success')
             return jsonify_data(html=_render_paper_dashboard(self.event))
         return jsonify_form(form)
+
+
+class RHManageReviewingQuestions(RHManagePapersBase):
+    def _process(self):
+        review_type = request.view_args['review_type']
+        if review_type == 'layout':
+            questions = self.event.cfp.layout_review_questions
+        else:
+            questions = self.event.cfp.content_review_questions
+
+        actions = {'create': 'papers.create_reviewing_question', 'edit': 'papers.edit_reviewing_question',
+                   'delete': 'papers.delete_reviewing_question', 'sort': 'papers.sort_reviewing_questions'}
+        args = {'create': {'review_type': review_type}, 'sort': {'review_type': review_type}}
```

Comment:
I don't think the nested dict is very useful here. I'd just go for `common_url_args = {'review_type': review_type}` and use that when building the URLs that are not for a specific question.